File: EUREKA_BANK_SOAP_DOTNET/Eurekabank_Servidor_SOAP/Eurekabank_Servidor_SOAP/Models/MovimientoRequest.cs
using System.Runtime.Serialization;

namespace Eurekabank_Servidor_SOAP.Models
{
    [DataContract]
    public class MovimientoRequest
    {
        [DataMember]
        public string CuentaOrigen { get; set; }

        [DataMember]
        public decimal Importe { get; set; }

        [DataMember]
        public string TipoMovimiento { get; set; }

        [DataMember]
        public string? CuentaDestino { get; set; }
    }
}

File: EUREKA_BANK_RESTFUL_JAVA_BDD_NUBE/02. CLIMOV/EUREKA_BANK_CLIMOV_RESTFUL2/app/src/main/java/ec/edu/eureka_bank_climov_restful/model/MovimientoRequest.java
package ec.edu.eureka_bank_climov_restful.model;

public class MovimientoRequest {

    private String cuentaOrigen;
    private String cuentaDestino;
    private String tipoMovimiento;
    private double importe;

    // Setter y getter para cuentaOrigen
    public String getCuentaOrigen() {
        return cuentaOrigen;
    }

    public void setCuentaOrigen(String cuentaOrigen) {
        this.cuentaOrigen = cuentaOrigen;
    }

    // Setter y getter para cuentaDestino
    public String getCuentaDestino() {
        return cuentaDestino;
    }

    public void setCuentaDestino(String cuentaDestino) {
        this.cuentaDestino = cuentaDestino;
    }

    // Setter y getter para tipoMovimiento
    public String getTipoMovimiento() {
        return tipoMovimiento;
    }

    public void setTipoMovimiento(String tipoMovimiento) {
        this.tipoMovimiento = tipoMovimiento;
    }

    // Setter y getter para importe
    public double getImporte() {
        return importe;
    }

    public void setImporte(double importe) {
        this.importe = importe;
    }
}
